coalib/settings/FunctionMetadata.py

```diff
@@ -91,6 +91,21 @@ def optional_params(self):
         """
         return self._filter_out_omitted(self._optional_params)
 
+    def add_alias(self, original, alias):
+        """
+        Adds an alias for the original setting. The alias setting will have
+        the same metadata as the original one. If the original setting is not
+        optional, the alias will default to ``None``.
+
+        :param original:  The name of the new setting.
```

Comment:
well this is the original, the other one is the alias for the original, it's not something new/old as this is not specific to deprecation. I'l change the doc content.